src/player_computer.c

```diff
@@ -211,7 +211,7 @@ struct ComputerTask *computer_setup_build_room(struct Computer2 *comp, RoomKind
         max_slabs = width_slabs;
     long area_min = (max_slabs + 1) / 2 + 1;
     long area_max = area_min / 3 + 2 * area_min;
-    const long arr_length = sizeof(look_through_rooms)/sizeof(look_through_rooms[0]);
+    const long arr_length = ROOM_TYPES_COUNT_OLD;
```

Comment:
game.conf.slab_conf.room_types_count would make more sense I think, and get rid of the OLD field

Reply:
> game.conf.slab_conf.room_types_count would make more sense I think, and get rid of the OLD field

Could the AI already build new rooms?

Reply:
probably would need more spots changed for that

Reply:
> probably would need more spots changed for that

right, so that is why I put this in, so that the AI only checks the rooms it can already build.